spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -38,6 +38,30 @@
       end
     end
 
+    context 'when the limit is reached' do
+      let!(:targets) { create_list(:target, Target::MAX_TARGET_LIMIT, user: user) }
```

Comment:
I don't know if it's appropriate here to create a list with the size of the constant, because the objective of these tests is to validate the requirements, so that if someone else changes code, the previously implemented requirements don't break. So, what should be tested here is what the requirement says, which is max 10 targets.

However I would wait for another opinion on this matter before changing anything. @sebaherrera07 @TimoPeraza , WDYT?

Reply:
In this case it is not hardcoded, because she is using the constant. If she had put `10` , then yes it would be wrong because if the constant changed, the test would break. But as she is using the same constant, the test would remain valid no matter if it changes.